src/SnackbarProvider/SnackbarProvider.tsx

```diff
@@ -255,14 +255,14 @@ class SnackbarProvider extends Component<SnackbarProviderProps, State> {
     /**
      * Close snackbar with the given key
      */
-    closeSnackbar: ProviderContext['closeSnackbar'] = (key) => {
+    closeSnackbar: ProviderContext['closeSnackbar'] = (key, reason = 'instructed') => {
```

Comment:
Reason has been set by default to 'instructed' to avoid any possible issues with migrating to next version.
this PR should be a minor change given that existing implementation don't change in behavior